stackdriver/Gemfile

```diff
@@ -0,0 +1,7 @@
+source "https://rubygems.org"
+
+gem "sinatra", "~> 1.4"
```

Comment:
Y?

Reply:
The integrations tests only cover 1.4 at the moment. I requested that Heng update google-cloud-ruby tests to include Sinatra 2.0 as it makes sense moving forward. 

Current tested versions:
https://github.com/GoogleCloudPlatform/google-cloud-ruby/tree/master/integration

Cur

Reply:
Tested the samples manually with Sinatra 2.0